api/e2e/gql_deployment_pagination_test.go

```
@@ -0,0 +1,446 @@
+		resp := e.POST("/api/graphql").
+			WithHeader("Content-Type", "application/json").
+			WithHeader("X-Reearth-Debug-User", uId1.String()).
+			WithBytes(jsonData).
+			Expect().Status(http.StatusOK)
+
+		var result struct {
+			Data struct {
+				Deployments struct {
+					Nodes []struct {
+						ID        string    `json:"id"`
+						updatedAt time.Time `json:"updatedAt"`
```

golangci-lint (1.62.2)

structtag: struct field updatedAt has json tag but is not exported

(govet)